pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1678,13 +1680,15 @@
         "1.3.0",
         "1.1.0",
         "1.2.0",
-        "1.4.1"
+        "1.4.1",
+        "1.4.2"
       ],
       "BaselineVersion": "1.4.1",
       "AssemblyVersionInPackageVersion": {
         "1.0.22.0": "1.0.22",
         "1.3.0.0": "1.3.0",
         "1.4.1.0": "1.4.1",
+        "1.4.1.0": "1.4.2",
```

Comment:
Is the right thing making version 1.4.1.0 show up in the 1.4.2. package since that will essentially be true?